src/data.js

```diff
@@ -78,6 +78,23 @@ const pages = [
       'Blogger',
     ],
   },
+  {
+  	name: 'Sam Cross',
+  	description: '"Developer", IT student, frequent traveller by coach, addicted to music.',
+  	url: 'https://sam-cross.github.io/uses',
```

Comment:
`404`

Reply:
Oops - that wasn't supposed to be in the PR. Didn't realise updating the master branch on my repo automatically added commits to this - sorry! I'll try to get that link fixed anyway.